Makefile

```diff
@@ -0,0 +1,19 @@
+# Build an executable for running go server after database migrations pass
+# 
+SHELL := /bin/bash
+
```

Comment:
Please remove all this bash stuff. You just need to have simple commands like `make migrate`, `make build`, `make test` etc.